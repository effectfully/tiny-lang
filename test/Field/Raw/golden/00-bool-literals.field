let ?false = F;
let ?true  = T;