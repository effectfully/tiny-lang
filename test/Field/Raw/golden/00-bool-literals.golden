Program (Statements [ELet (Var "?false") (EConst (Some (UniConst Bool False))),ELet (Var "?true") (EConst (Some (UniConst Bool True)))])